README.md

```diff
@@ -47,6 +58,11 @@ var Panda = function(name, age) {
 Your Answer:
 ```js
 // your code here
+var value = parseInt($('#eat_bamboo').html());
+  value++;
+  $(#eat_bamboo).html(value);
+} 1);
```

Comment:
Remember, you are free to try the code in the console to verify.  Please review our recommendation in the solution branch.